File: app/src/main/java/com/gystry/pjetpack/utils/StringConvert.java
package com.gystry.pjetpack.utils;

/**
 * @author gystry
 * 创建日期：2020/7/8 14
 * 邮箱：gystry@163.com
 * 描述：
 */
public class StringConvert {
    public static String convertFeedUgc(int count) {
        if (count < 10000) {
            return String.valueOf(count);
        }
        return count / 10000 + "万";
    }
}

File: appkt/src/main/java/com/gystry/appkt/utils/StringConvert.kt
package com.gystry.appkt.utils

/**
 * @author gystry
 * 创建日期：2021/3/30 14
 * 邮箱：gystry@163.com
 * 描述：
 */
fun convertFeedUgc(count:Int):String{
    if(count<10000){
        return count.toString()
    }
    return "${count/10000}万"
}

fun convertTagFeedList(num:Int):String{
    return if(num<10000){
        "${num}人观看"
    }else{
        "${num/10000}万人观看"
    }
}